### PR TITLE
MINOR: example.com moved

### PR DESCRIPTION
**Pull request by akatona84**
### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Review by viktorsomogyi — APPROVED** [review]

**Comment by akatona84** [comment]
now the ipv6 address is changed to 2606:2800:220:1:248:1893:25c8:1946...

**Comment by akatona84** [comment]
ipv6 alternates between 2606:2800:21f:cb07:6820:80da:af6b:8b2c and the old one, it's hard to fix the test like this.

File: clients/src/test/java/org/apache/kafka/clients/ClientUtilsTest.java [hunk 1]
```diff
@@ -61,7 +61,7 @@ public void testParseAndValidateAddressesWithReverseLookup() {
         assertTrue(validatedAddresses.size() >= 1, "Unexpected addresses " + validatedAddresses);
         List<String> validatedHostNames = validatedAddresses.stream().map(InetSocketAddress::getHostName)
                 .collect(Collectors.toList());
-        List<String> expectedHostNames = asList("93.184.216.34", "2606:2800:220:1:248:1893:25c8:1946");
+        List<String> expectedHostNames = asList("93.184.215.14", "2606:2800:220:1:248:1893:25c8:1946");
```
Comment (chia7712) [inline clients/src/test/java/org/apache/kafka/clients/ClientUtilsTest.java]: That is weird. What I see on my local is `93.184.215.14, 2606:2800:21f:cb07:6820:80da:af6b:8b2c`.

**Review by chia7712 — COMMENTED** [review]
@akatona84 thanks for quick fix. the address resolved on my local is different...

File: clients/src/test/java/org/apache/kafka/clients/ClientUtilsTest.java (hunk 1 above)
Comment (akatona84) [reply clients/src/test/java/org/apache/kafka/clients/ClientUtilsTest.java]: Yeah, it's changing back n forth.

**Review by brandboat — APPROVED** [review]
LGTM, just encountered the same issue.

File: clients/src/test/java/org/apache/kafka/clients/ClientUtilsTest.java (hunk 1 above)
Comment (brandboat) [reply clients/src/test/java/org/apache/kafka/clients/ClientUtilsTest.java]: Sorry, I read too quickly. Indeed, the IPv6 address I saw in tests is also 2606:2800:21f:cb07:6820:80da:af6b:8b2c

**Comment by viktorsomogyi** [comment]
Ok, a couple of tests failed but none of them is ClientUtilsTest, so I'm merging this. Thank you @akatona84 for your contribution, @brandboat and @chia7712 for the reviews.

**Comment by omkreddy** [comment]
Lets also cherry-pick to 3.7 and 3.6 branches

**Comment by viktorsomogyi** [comment]
Yea, just had the same thought, will do in a couple of minutes.

**Pull request merged**